sensing/pointcloud_preprocessor/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ ament_auto_add_library(pointcloud_preprocessor_filter SHARED
   src/concatenate_data/concatenate_data_nodelet.cpp
   src/crop_box_filter/crop_box_filter_nodelet.cpp
   src/downsample_filter/voxel_grid_downsample_filter_nodelet.cpp
+  src/downsample_filter/faster_voxel_grid_downsample_filter.cpp
```

Comment:
It would be great if they could be defined and combined as separate libraries.
Related link: https://github.com/autowarefoundation/autoware.universe/pull/3679/files#r1195848230

Reply:
Addressed in https://github.com/autowarefoundation/autoware.universe/pull/3679/commits/6b859ec9534f4bea93d5c58c4781be8be306fe98.